README.md

```diff
@@ -177,44 +177,39 @@ Example of a partial CR:
 
 ### Mounting and injecting binding information
 
-This specification allows for data to be mounted using volumes or injected using environment variables.  The best practice is to mount any sensitive information, such as passwords, since that will avoid accidentally exposure via environment dumps and subprocesses.  Also, binding binary data (e.g. .p12 certificate for Kafka) as an environment variable might cause a pod to fail to start (stuck on `CrashLoopBackOff`), so it advisable for backing services with such binding data to mark it with `bindAs: volume`.
+#### Mounting data
+Implementations of this specification **MUST** mount the binding data into the consuming application container in the following location:
+
+```
+<mountPathPrefix>/<service-id>/secret/<persisted_secret>
+```
+
+Where:
+* `<mountPathPrefix>` defaults to `/platform/bindings` if not specified in the `ServiceBinding` CR via the `mountPathPrefix` element.  The environment variable `SERVICE_BINDINGS` **MUST** be set to the value of `<mountPathPrefix>`, so that applications can always find this directory.
```

Comment:
would it nice if we rename the `SERVICE_BINDINGS` to `SERVICE_BINDINGS_MOUNT_PATH` or something similar which better implies the intent and meaning?

Reply:
good point @yharish991 - how about `SERVICE_BINDINGS_ROOT`?